### PR TITLE
Added new featured guardian

### LINKED ISSUE #115: "<br />" tags visible in Featured Guardian Q&A section
screen shot 2016-09-01 at 12 18 22 pm

### LINKED ISSUE #117: Add mention that stats apply to players active within the last 6 months

### LINKED ISSUE #118: Give props to guardian.gg for PvP-related stats

### PR DESCRIPTION
**Pull request by john-mikhail**
Need to crop the image a little to make it the same size as previous.  Will work on that at home.

Fixes #118 
Fixes #117
Fixes #115

### CODE REVIEW

**Comment by john-mikhail** [comment]
Fixes #118 
Fixes #117

File: public/elements/zero-featured-guardian.html [hunk 1]
```diff
@@ -121,7 +121,7 @@
       },
 
       _toggle: function() {
-        var moreInfo = document.getElementById('guardian-qa');
+        var moreInfo = document.getElementById('guardian-qa-' + this.guardian);
```
Comment (pylaligand) [inline public/elements/zero-featured-guardian.html]: I assume that will be useful for the "past featured guardians" page.

File: public/elements/zero-stats.html [hunk 2]
```diff
@@ -45,6 +45,8 @@
     <p>
       Want to compare yourself to your fellow Guardians?  Well you've come to the right place.  Select a stat to
       get a more comprehensive view and see where you stack rank.
+      <br />
+      Please note that stats apply to active clan members within the last 6 months.
```
Comment (pylaligand) [inline public/elements/zero-stats.html]: "to clan members active within the last 6 months"?

File: public/elements/zero-featured-guardian.html (hunk 1 above)
Comment (john-mikhail) [inline public/elements/zero-featured-guardian.html]: Yea this was to fix a bug where adding more than one past guardian, the IDs were not unique.  So toggling one section opened them all up.

File: public/elements/zero-stats.html (hunk 2 above)
Comment (john-mikhail) [inline public/elements/zero-stats.html]: Done.

**Pull request merged**